docs/reference/authentication.md

```diff
@@ -27,6 +27,17 @@ Web UI at Administration / Users to create users.  Users have an access key
 `AKIA...` and an associated secret access key.  These credentials are valid
 to log into the Web UI, or to authenticate programmatic requests to the API
 Server or the S3 Gateway.
+git
+#### lakeFS Invite
+
+If you were invited to lakeFS, Follow these steps to create and activate your account:
```

Comment:
As mentioned in this review, please correct spelling and capitalization.

Reply:
Done.